test.js

```diff
@@ -1 +1,10 @@
-console.log('Hello World')
+// the hello world program
+console.log('Hello World');
+
+console.log('Hello Universe');
+
+console.log('Hello other stuff');
+
+if(foo === "bar") {
+  console.log('stuff')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/troubleshoot-codacy/no-suggestions-2/pullRequest?prid=8692777)
```suggestion
  console.log("stuff")
```